quantum/snippets.js

```diff
@@ -37,6 +37,14 @@ const BARRIER =
 const HADAMARD_GATE =
 `qc.h(%s)`;
 
+const MEASUREMENT =
+  `qc.measure(%s)`;
+
+const SIMULATOR = `simulator = Aer.get_backend('qasm_simulator')
+result = execute(qc, backend = simulator, shots = %s).result()
+counts = result.get_counts()
+print(counts)`;
+
```

Comment:
```suggestion
const MEASUREMENT =
`qc.measure(%s)`;

const SIMULATOR =
`simulator = Aer.get_backend('qasm_simulator')
result = execute(qc, backend = simulator, shots = %s).result()
counts = result.get_counts()
print(counts)`;

```
Small modification to this just to make it consistent/easier to read.